libqtile/backend/wayland/core.py

```diff
@@ -1286,26 +1287,19 @@ def unhide_cursor(self) -> None:
     @expose_command()
     def get_inputs(self) -> dict[str, list[dict[str, str]]]:
         """Get information on all input devices."""
-        info = {}
-        device_lists: dict[str, list[inputs._Device]] = {
-            "type:keyboard": self.keyboards,  # type: ignore
-            "type:pointer": self._pointers,  # type: ignore
-        }
-
-        for type_key, devices in device_lists.items():
-            type_info = []
-
-            for dev in devices:
-                type_info.append(
-                    dict(
-                        name=dev.wlr_device.name,
-                        identifier=dev.get_info()[1],
-                    )
-                )
-
-            info[type_key] = type_info
+        info: DefaultDict[str, list[dict]] = defaultdict(lambda: [])
```

Comment:
We no longer need to use typing.DefaultDict for annotations, and I think we can avoid creating a new object (the lambda) by just using list:
```suggestion
        info: defaultdict[str, list[dict]] = defaultdict(list)
```

Reply:
Ah I need to read up on the updated annotations. Thanks!